website/common/locales/en/tasks.json

```diff
@@ -1,7 +1,7 @@
 {
     "clearCompleted": "Delete Completed",
     "lotOfToDos": "Your most recent 30 completed To-Dos are shown here. You can see older completed To-Dos from Data > Data Display Tool or Data > Export Data > User Data.",
-    "deleteToDosExplanation": "If you click the button below, all of your completed To-Dos and archived To-Dos will be permanently deleted. Export them first if you want to keep a record of them.",
+    "deleteToDosExplanation": "Permanently delete all of your completed and archived To-Dos. Except for To-Dos from active challenges and Groups Plans. Export them first if you want to keep a record of them.",
```

Comment:
@CutTheCode I'm not sure if you saw my comment here. This still needs to be changed before the PR can be merged.

Also can you put your full User ID into the top post? You can find it at Settings > API on the website.